share/man/man5/intro.5

```diff
@@ -25,18 +28,32 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd February 16, 1997
+.Dd November 17, 2024
 .Dt INTRO 5
 .Os
 .Sh NAME
 .Nm intro
-.Nd "introduction to file formats"
+.Nd introduction to file formats
 .Sh DESCRIPTION
-This section contains information about file formats.
+This section contains information about the file formats
+which comprise most data structures in the
+.Bx
+environment, including:
+.Pp
+.Bl -bullet -compact
+.It
+ASCII configuration and resource files
+.It
+System binary file and stream structures
+.It
+Composition of database files
+.El
 .Sh FILES
-.Bl -tag -width /etc/shells -compact
-.It Pa /etc
-location of most system configuration files
+.Bl -tag -width "/usr/local/etc/" -compact
+.It Pa /etc/
+location of most base system configuration files
+.It Pa /usr/local/etc/
+location of installed component configuration files
```

Comment:
i appreciate there's a desire to be terse here, but this wording seems a bit odd to me.

for unfortunate historical reasons, `/usr/local/etc` contains two separate, unrelated types of file:

- user-created configuration files for locally-installed software
- configuration files for software installed from FreeBSD ports/packages

however:

- not every 'installed component' is guaranteed to go to `/usr/local/etc` (e.g. pkgsrc, OpenPKG)
- the installation prefix of ports (`LOCALBASE`) can be changed when building them

perhaps this could be written as something like:

> typical location of configuration files for locally-installed software, and default location of configuration files for software installed from FreeBSD ports

(or, you know, something less awkward -- there's a reason i'm not a doc author.)

and an Xr to `hier(7)` might not hurt there to explain `/usr/local` more generally.

Reply:
You are correct in your assessment, and in general I agree. However I believe we should not be overly-hospitable when it comes to enumerating all possible edge-cases that differ from the default/general case; eventually it will harm the clarity of the documentation.

For example, it is important that we maintain the functionality of `LOCALBASE` within all of our system tools and infrastructure. But you will find many man pages simply use a prefix of `/usr/local/`, as this is the default value. It should be understood by the intermediate user who wishes to tweak `LOCALBASE` (for whatever reason) that this translation is their responsibility.

Same goes for pkgsrc or openpkg (I personally had never heard of the latter before). These are supported use-cases, but not preferred or advertised in the way that `pkg(8)` is the standard method (or Tier-1 supported by FreeBSD). We cannot fully account for their usage in our documentation.

That said, here is my suggested wording for this particular case:
```
default location of configuration files for locally installed software (e.g. via pkg(7))
```

I have to say the descriptions in `hier(7)` are not particularly illuminating w.r.t. any of the alternative use-cases you brought up.

Reply:
I like mhorne's suggestion.

Reply:
I am willing to make this change, however I think this is a *lot* of redundancy, which is less straightforward and thus can be confusing, or sound less authorative. I don't know if I'm explaining it right but parenthesis drops the readers confidence interval.

I agree with the "not being overly hospitable arguement". Every single thing can be configured to do absolutely whatever you want even if it won't even work that way.

The design is that cfg for base software goes in /etc/, and cfg for installed software goes /usr/local/etc/.

Does 50 years of unix with now over 40k pkgs have a few weird corner cases that defy any attempt to make a terse, precise statement about how it's designed? Of course.

But does that mean we shouldn't do it?

Let me chew on this a bit more.